templates/block_creator/block_creator.py

```diff
@@ -86,6 +87,36 @@ def _get_container(self):
 
         return self.api.services.find_or_create(CONTAINER_TEMPLATE_UID, self._container_name, data=container_data)
 
+    @retry((RuntimeError), tries=5, delay=2, backoff=2)
+    def _wallet_init(self):
+        """
+        initialize the wallet with a new seed and password
+        """
+        try:
+            self.state.check('wallet', 'init', 'ok')
+            return
+        except StateCheckError:
+            pass
+
+        self.logger.info('initializing wallet %s', self.name)
+        self._client_sal.wallet_init()
```

Comment:
What will happen when the wallet was already there?
Will it wipe it?

Reply:
It will fails, cause tfchainc returns an error when trying to init twice the same wallet

Reply:
Ok, thats exactly the behavior i would want here.